spec/std/yaml/to_yaml_spec.cr

```diff
@@ -0,0 +1,36 @@
+require "spec"
+require "yaml"
+
+describe "String.to_yaml" do
```

Comment:
I'd also suggest to separate this like described in [comment](https://github.com/crystal-lang/crystal/pull/4838/files#r137747782)